setup.py

```diff
@@ -44,7 +44,14 @@
         "onnxruntime<1.15.0",
         "transformers>=4.34.0",
     ],
-    "openvino": ["openvino>=2023.2", "onnx", "onnxruntime", "transformers>=4.36.0", "optimum>=1.16.1"],
+    "openvino": [
+        "openvino>=2023.3",
+        "onnx",
+        "onnxruntime",
+        "transformers>=4.36.0",
+        "optimum>=1.16.1",
+        "openvino-tokenizers[transformers]",
```

Comment:
Per our agreement with HF team, we should keep it as a separate extra "openvino-tokenizers". @apaniukov, can you please change it accordingly?

Reply:
Done.